### PR TITLE
Initial build

### PR DESCRIPTION
**Pull request by ewen**
A starting point for a Media Suite Django project.

Aims to get you up and running with a working instance that will run in both Docker and using the Django development server.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:16.04
+
+MAINTAINER Media Suite
+
+# Install base software into the image.
+# Pulls from Ubuntu repos and then cleans up the package lists to save some space.
+# Last step upgrades pip (using pip) to the latest version.
+RUN \
+  apt-get update && \
+  apt-get upgrade -y && \
```
Comment (tfentonz) [inline Dockerfile]: From [Best practices for writing Dockerfiles](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/)  you should avoid `RUN apt-get upgrade`.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    python3 \
+    python3-pip \
+    nginx \
+    supervisor \
+    && \
+  rm -rf /var/lib/apt/lists/* && \
+  pip3 install -U pip
+
+# Deployment specific packages
+RUN pip3 install uwsgi
```
Comment (tfentonz) [inline Dockerfile]: Can this be chained to the previous `RUN`?

**Comment by tfentonz** [comment]
I get the following when I try and build the docker image due to missing `server/manage.py`. I might be on the wrong track here as this might just a template for a project.

```
Step 12/14 : RUN SECRET_KEY='x' python3 /src/server/manage.py collectstatic --no-input
 ---> Running in ea04f67ba9c7
python3: can't open file '/src/server/manage.py': [Errno 2] No such file or directory
The command '/bin/sh -c SECRET_KEY='x' python3 /src/server/manage.py collectstatic --no-input' 
returned a non-zero code: 2
```

Do you also need a `.dockerignore`?

File: Dockerfile (hunk 2 above)
Comment (ewen) [reply Dockerfile]: Yep

**Comment by ewen** [comment]
It should build as expected but you'll need to follow the steps prior to the docker build in the README.

Good call on `.dockerignore`.

**Pull request merged**